### DISCUSSION TITLE
Alleged bug: regex whitelist entry does not turn off trusted-click-element scriptlet

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#3046] [General]
@krystian3w 

> For me regex in trusted list (of domains) don't/can't disable `trusted-click-element` (rest scriptlets may disabled to run), but I can't create minimal online demo without share company app with private data.
> 
> Reinstall then does not help.

***

> 
> I can't cleanup webpage to share minimal demo compatible with any PC, I guess gorhill reject video with dynamic added blur on private data/logo+URL.
> 
> Also I don't want itself found bug in Browser or addon code.

---

> > please create issue or discussion with steps to reproduce.
> 
> - I create regex similar to:
> `/\/(index\.php)?[\w=]+(\?|&)param=value_name_long/`
> - I create clicker to company webpapp based on `trusted-click-element`
> - I open page excluded by regex (icon is gray)
> - almost filters was not excecuted but `trusted-click-element` does

***

Maybe similar: https://github.com/uBlockOrigin/uBlock-issues/issues/3083

**Comment by ghost** [comment]
Chrome / Firefox?

Troubleshooting info?

Isn't it inside iframe? (maybe bug in recent change to click inside iframes)

I recall some bug about not disabling scriptlets on whitelisted pages... https://github.com/uBlockOrigin/uBlock-issues/issues/2896

**Reply by MasterKia** [reply]
> I recall some bug about not disabling scriptlets on whitelisted pages...

That's uBOL, not uBO:
https://github.com/uBlockOrigin/uBOL-home/issues/20#issuecomment-1358025196

**Reply by krystian3w** [reply]
But I do not use `trusted-click-element` in uBO-L (I do not have time to manipulate the file with compiled filters manually) but in uBO Dev MV2.

Also uBO-L may reject these regex exceptions: `/\/(index\.php)?[\w=]+(\?|&)param=value_name_long/` in Browser panel.

**Comment by krystian3w** [comment]
> Chrome / Firefox?

IMO both, so bug can by in both (so in addon code or API implementation)


> Troubleshooting info?

IMO also on stock so not releated with too many filters

> Isn't it inside iframe? (maybe bug in recent change to click inside iframes)

Page don't use frames

**Comment by krystian3w** [comment]
I tried blur GDPR/company data at page in postproduction and made something like:

https://sendvid.com/8ydepkcp (~445 in order to publish)

Click action during after 30 second should not triggered by uBo MV2.

If I add more simple domain exclusion (wildcard/path without regex) the click actions has stopped.

I think to create minimal demo we need two small pages with open 'itself' action second in loop on click.

**Reply by MasterKia** [reply]
You need to use a translator.

**Reply by krystian3w** [reply]
Content in video can be in Chinese - the first scriptlet (from video) should not be allowed to click on a button resembling a checklist with items checked off by added exception in Trusted list (regex).

**Comment by krystian3w** [comment]
Possible add more scriptlets to title:

- `aopr` (maybe too `aopw`)
- legacy `remove-attr`
- `acs` if we don't have race conditions
- `set-attr`

These do not helps for Firefox 115.7 https://github.com/gorhill/uBlock/commit/13dcd844a7d510fe5fae7dcd71c5a908e38b3195 (but can helps in Chrome 109)
